src/components/NavigationBar/index.js

```diff
@@ -669,24 +669,30 @@ class NavigationBar extends Component {
                         {isMobileView(400) ? (
                           <Add
                             style={{
-                              marginLeft: '5px',
+                              marginLeft: '0px',
+                              marginRight: '0px',
+                              marginTop: '3px',
                               color: '#fff',
                             }}
                           />
                         ) : (
-                          <CreateDetail>Create</CreateDetail>
+                          <CreateDetail style={{ padding: '7px' }}>
+                            Create
+                          </CreateDetail>
                         )}
                       </div>
                     </StyledIconButton>
-                    <ToolTip title="Dashboard">
-                      <IconButton
-                        color="inherit"
-                        onClick={() => history.push('/dashboard')}
-                        style={{ padding: '7px' }}
-                      >
-                        <Dashboard />
-                      </IconButton>
-                    </ToolTip>
+                    {!isMobileView(400) ? (
```

Comment:
Can you store the value of `isMobileView(400)` in a variable and then use it at instances to avoid multiple function calls?